core/src/main/java/amino/run/oms/metric/MicroServiceMetric.java

```diff
@@ -0,0 +1,64 @@
+package amino.run.oms.metric;
+
+import amino.run.common.ReplicaID;
+import amino.run.kernel.metric.RPCMetric;
+import java.net.InetSocketAddress;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.logging.Logger;
+
+/**
+ * MicroService metric class maintains the remote procedure call metrics for the replica with
+ * respect to each client those RPCs are received from.
+ */
+public class MicroServiceMetric {
```

Comment:
This looks like essentially a cut 'n paste of KernelServerMetric class above?
Who do this rather than use inheritance or some other better code reuse mechanism than cut 'n paste?